homeassistant/components/insteon_hub.py

```diff
@@ -50,6 +49,17 @@ def setup(hass, config):
         _LOGGER.error("Could not connect to Insteon service")
         return False
 
-    discovery.load_platform(hass, 'light', DOMAIN, {}, config)
+    discovery.load_platform(
+        hass,
+        'light',
+        DOMAIN,
+        {CONF_POLL: config[DOMAIN][CONF_POLL]},
+        config
+    )
+    for insteon_house in INSTEON.houses:
+        threading.Thread(
```

Comment:
There is a catch all exception handler in the stream method in `insteon-hub` that closes the stream.